README.md

```diff
@@ -207,31 +162,131 @@ Users want to determine several types of the formatting width as following
   + fr-FR
     - 1 heure, 46 minutes et 40 secondes
 
-### Design
+#### Design
+
+Adding the locale in string format as a first argument, or specifying a ranked list of locales as an array of string locale values.
+
+### Display fractional values
+
+Sometimes it is desirable to display the smallest sub-second unit not by itself but as a fraction of the immediately larger unit.
 
-Adding the locale in `string` format as a first argument.
+#### Design
 
-# API design
+We allow users to specify a `fractionalDigits` option that will display the smallest sub-second unit with display set to `"auto"` as a fraction of the previous unit if it is non-zero. The number of digits used will be the value passed to this option.
+
+#### Example
+
+```javascript
+new Intl.DurationFormat('en', { fractionalDigits: 2 }).format('PT12.3456S'); // => 12.34 sec
+new Intl.DurationFormat('en', { milliseconds: 'narrow', fractionalDigits: 2 }).format('PT12.3456S'); // => 12 sec 345.60 ms
```

Comment:
```suggestion
new Intl.DurationFormat('en', { milliseconds: 'narrow', fractionalDigits: 2 }).format('PT12.3456S'); // => 12s 345.60ms
```
Is this the correct `'narrow'` output?

Reply:
About this and the comment above, I'm not entirely sure. I think it's still not exactly confirmed yet since the locale data is not finalized. At the end of the day, this is just a "sample" output for demonstration and the exact result will be decided by (and could be changed in the future by) linguists at the Unicode/CLDR side of things.

Reply:
Yep, agreed. Mostly I think that this doc, for clarity, should be internally consistent in how "narrow" behaves. We giver several examples of narrow in the doc. Their output should match.

Also, I'd really like to avoid a space between the number and a single letter like "8 d". It looks really confusing when multiple units are shown, e.g. "4 y 3 m 6 d", because the same spacing is used between units as between units and label.

Reply:
Sounds fair, I'd replace all occurrences with the latter format.

Reply:
Actually, in this specific case, I'd just miswritten the `"short"` output instead of `"narrow"`.